src/backend/brokers/go.emails/broker.go

```diff
@@ -68,6 +68,15 @@ func Initialize(conf LDAConfig) (broker *EmailBroker, connectors EmailBrokerConn
 			Hosts:       conf.StoreConfig.Hosts,
 			Keyspace:    conf.StoreConfig.Keyspace,
 			Consistency: gocql.Consistency(conf.StoreConfig.Consistency),
+			SizeLimit:   conf.StoreConfig.SizeLimit,
+		}
+		if conf.S3Service == "minio" {
```

Comment:
this configuration apply to any s3 object store, minio or another.
Don't set explicit reference to minio